ipassign/protocol/payload.py

```diff
@@ -186,7 +187,7 @@ def __str__(self):
             flags += ' dynamic'
         if self.flash:
             flags += ' flash'
-        ret = f"""[payload]
+        ret = f"""[configuration]
```

Comment:
For clarity in prints (eg in `ipassing-listener`), it would be nicer to keep the `[payload]` header, and add a field that specifies what type of payload it is.